app/views/modules/_access_control.html.erb

```diff
@@ -64,6 +64,37 @@ Unless required by applicable law or agreed to in writing, software distributed
   </div>
 </div>
 
+<div class="card item-lending-period">
+  <div class="card-header">
+    <h3 class="card-title">Item lending period</h3>
+  </div>
+  <div class="card-body">
+    <div class="form-group">
+      <% field = object.is_a?(Admin::Collection) ? :default_lending_period : :lending_period %>
```

Comment:
I think the approach taken for the other fields like visibility is to use a common name in the view partial and do the mapping in the call to render.  This could avoid the branching logic.